### DISCUSSION TITLE
How can I change the properties of wind or turn off the engine in Python

### DISCUSSION THREAD

**Discussion by dyoung666** [JSBSim-Team/jsbsim#764] [General]
How can I change the properties of wind or turn off the engine in Python

**Comment by bcoconni** [comment]
Wind can be modified using the properties `atmosphere/psiw-rad` (wind direction) and `atmosphere/wind-mag-fps` (wind velocity).

The property to turn an engine off depends on the type of the engine you are using:
* For turbine engines, set the property `propulsion/cutoff_cmd` to 1.
* For piston engines, set the property `propulsion/magneto_cmd` to 0.
